e2e/hardhatTransactionFlow.spec.js

```diff
@@ -206,19 +253,36 @@ describe('Hardhat Transaction Flow', () => {
     await Helpers.tapAndLongPress('send-confirmation-button');
     await Helpers.checkIfVisible('profile-screen');
     await Helpers.swipe('profile-screen', 'left', 'slow');
+    const postSendBalance = await getOnchainBalance(
+      RAINBOW_WALLET_DOT_ETH,
+      BAT_TOKEN_ADDRESS
+    );
+    if (!postSendBalance.gt(preSendBalance))
```

Comment:
because we're still using jest we should do something like this:

```
const jestExpect = require('expect');
...
jestExpect(postSendBalance.gt(preSendBalance)).toBe(true);
```

Reply:
Not 100% sure but you might have to turn off  initGlobals first so detox doesn't override `expect`

`detox.init(config, {initGlobals: false});`

and change the Helpers `expect` to `detox.expect`